src/utils.cpp

```diff
@@ -20,6 +20,7 @@
  */
 
 #include "mega/utils.h"
+#include "mega/base64.h"
```

Comment:
We don't need bases64 here if the `pubk` parameter is in a binary format already.